readthedocs_theme/templates/base.html

```diff
@@ -61,6 +61,14 @@
 <body>
   {% include 'includes/topnav.html' %}
 
+  <div class="ui center aligned header">
+        {# TODO: Replace with live URL #}
+        <a href="https://readthedocs-blog--212.org.readthedocs.build/website-migration/">
+        <i class="fad fa-rocket-launch icon"></i>
+        Welcome to our new website! 
```

Comment:
I didn't think to click this heading. It would help to find some supporting text/subheading that mentions the blog post or something similar.

Reply:
Yea, this is just an example. Thoughts on what would work best here in SUI?

Reply:
I mentioned above using `.ui.message`, though I tried this and it took up a lot of screen real estate with an icon, header, and supporting line below. I did a hybrid approach with 1be48d7